examples/ruby/README.md

```diff
@@ -1,3 +1,7 @@
+**API `v1` is deprecated**
+
+***
+
```

Comment:
Should we link to these?
https://github.com/planningcenter/pco_api_ruby
https://github.com/planningcenter/pco_api_oauth_example